app.rambox.ramboxce.yaml

```diff
@@ -0,0 +1,88 @@
+app-id: app.rambox.ramboxce
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+default-branch: stable
+#add-extensions:
+#  org.freedesktop.Platform.Compat.i386:
+#    directory: lib/i386-linux-gnu
+#    version: '21.08'
+#  org.freedesktop.Platform.Compat.i386.Debug:
+#    directory: lib/debug/lib/i386-linux-gnu
+#    version: '21.08'
+#    no-autodownload: true
+#  org.freedesktop.Platform.GL32:
+#    directory: lib/i386-linux-gnu/GL
+#    version: '1.4'
+#    versions: 21.08;1.4
+#    subdirectories: true
+#    no-autodownload: true
+#    autodelete: false
+#    add-ld-path: lib
+#    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d;vulkan/implicit_layer.d
+#    download-if: active-gl-driver
+#    enable-if: active-gl-driver
+
+sdk: org.freedesktop.Sdk
+#sdk-extensions:
+#  - org.freedesktop.Sdk.Compat.i386
+#  - org.freedesktop.Sdk.Extension.toolchain-i386
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+#  - --allow=multiarch
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --filesystem=xdg-config/kdeglobals:ro
+  - --filesystem=xdg-download
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.AppMenu.Registrar.*
+# - --system-talk-name=org.freedesktop.login1
+command: ramboxce
+tags: [opensource]
+desktop-file-name-suffix: ' (Flatpak)'
+separate-locales: false
+modules:
+
+  - shared-modules/libsecret/libsecret.json
```

Comment:
You have added entire shared-modules directory directly to git, instead of using it as a git submodule.